### PR TITLE
Remove STR_VIEWPORT

### PR DESCRIPTION
**Pull request by AaronVanGeffen**
This removes `STR_VIEWPORT` from StringIds.h. It's used in two places, but not for any particularly unique purpose.

I've replaced the test in `WidgetText` with one on `kWidgetContentEmpty`, a new constant I've introduced in Widget.h, just set to `0xFFFF'FFFF`.

### CODE REVIEW

**Review by tupaschoal — APPROVED** [review]

**Review by duncanspumpkin — APPROVED** [review]

File: src/openrct2-ui/interface/Widget.h [hunk 1]
```diff
@@ -19,7 +19,10 @@ ImageId GetColourButtonImage(colour_t colour);
 Widget* GetWidgetByIndex(const WindowBase& w, WidgetIndex widgetIndex);
 
 constexpr auto kWidgetsEnd = Widget{ WindowWidgetType::Last, 0, 0, 0, 0, 0, 0, 0 };
+constexpr uint32_t kWidgetContentEmpty = 0xFFFF'FFFF;
```
Comment (Gymnasiast) [inline src/openrct2-ui/interface/Widget.h:22]: We don’t use this notation anywhere in the codebase. We could of course decide to change it, but that would something for the team chat, I’d say.

Comment (AaronVanGeffen) [reply src/openrct2-ui/interface/Widget.h:22]: You mean the apostrophe to separate digits? It's a C++17 thing, I believe. Considering we started out with C, it's not that surprising the codebase doesn't use it yet.

If you believe its introduction warrants discussion, please raise it in the team chat, as suggested.

Comment (Gymnasiast) [reply src/openrct2-ui/interface/Widget.h:22]: Yes, I believe it does.

**Review by Gymnasiast — CHANGES_REQUESTED** [review]
I’d like to see the apostrophe taken out, or otherwise await the discussion in the team chat.

**Review by Gymnasiast — APPROVED** [review]

**Comment by AaronVanGeffen** [comment]
I have amended the commit to omit the apostrophe. Personally, I am convinced these digit-separating apostrophes vastly improve readability, though. Provided the discussion in team chat will move in favour of them, I will gladly introduce them consistently throughout the codebase.

But yes, let's focus this PR on STR_VIEWPORT's removal only.

**Pull request merged**